crates/telio-traversal/src/session_keeper.rs

```diff
@@ -62,7 +63,10 @@ pub struct SessionKeeper {
 }
 
 impl SessionKeeper {
-    pub fn start(sock_pool: Arc<SocketPool>) -> Result<Self> {
+    pub fn start(
+        sock_pool: Arc<SocketPool>,
+        activity_recorder: Option<Arc<dyn ActivityRecorder>>,
```

Comment:
I would like not to introduce an interface just for one function which is not gonna be used anywhere else, however I failed to extract it to a lambda. Suggestions very welcome :)